sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -443,6 +446,31 @@ case class AlterTableDropPartitionCommand(
 case class AlterTableRecoverPartitionsCommand(
     tableName: TableIdentifier,
     cmd: String = "ALTER TABLE RECOVER PARTITIONS") extends RunnableCommand {
+
+  // These are two fast stats in Hive Metastore
+  // see https://github.com/apache/hive/blob/master/
+  //   common/src/java/org/apache/hadoop/hive/common/StatsSetupConst.java#L88
```

Comment:
this is actually not a stable identifier (i.e. line 88 will likely change quickly) -- maybe just point to the file.